addons/im_livechat/static/src/legacy/public_livechat.js

```diff
@@ -177,21 +177,22 @@ var LivechatButton = Widget.extend({
      * @param {Array} notification
      */
     _handleNotification: function (notification) {
-        if (this._livechat && (notification[0] === this._livechat.getUUID())) {
-            if (notification[1]._type === 'history_command') { // history request
-                var cookie = utils.get_cookie(LIVECHAT_COOKIE_HISTORY);
-                var history = cookie ? JSON.parse(cookie) : [];
+        const [livechatUUID, notificationData] = notification;
+        if (this._livechat && (livechatUUID === this._livechat.getUUID())) {
+            if (notificationData._type === 'history_command') { // history request
+                const cookie = utils.get_cookie(LIVECHAT_COOKIE_HISTORY);
+                const history = cookie ? JSON.parse(cookie) : [];
                 session.rpc('/im_livechat/history', {
                     pid: this._livechat.getOperatorPID()[0],
                     channel_uuid: this._livechat.getUUID(),
                     page_history: history,
                 });
-            } else { // normal message
+            } else if (notificationData.id) { // normal message
```

Comment:
This condition is really weak. If changing the server is not considerable here, the condition should at least be complemented with checking the presence of some message specific param (maybe `body`).

Reply:
Don't know how to change server by the way so I'll just add a check on body